qemu/tests/balloon_hotplug.py

```diff
@@ -111,34 +107,28 @@ def enable_balloon_service():
 
         error_context.context("Unplug balloon device for %d times" % (i+1),
                               logging.info)
+
         out = new_dev.unplug(vm.monitor)
```

Comment:
Is it intended to keep device.unplug() here? Since you use simple_hotplug(), the device is inserted into vm.devices, so have to remove it after unplug by yourself if not using simple_unplug().

Reply:
@vivianQizhu  Yes, it is intened to use unplug() instead of simple_unplug.  If use simple_unplug, would hit an issue when unplug after migration:
01:46:43 ERROR|     out = vm.devices.simple_unplug(new_dev, vm.monitor)
01:46:43 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-62.0-py2.7.egg/virttest/qemu_devices/qcontainer.py", line 644, in simple_unplug
01:46:43 ERROR|     device = self[device]
01:46:43 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-62.0-py2.7.egg/virttest/qemu_devices/qcontainer.py", line 136, in __getitem__
01:46:43 ERROR|     raise KeyError("Device %s is not in %s" % (item, self))
01:46:43 ERROR| KeyError: 'Device balloon0 is not in <virttest.qemu_devices.qcontainer.DevContainer object at 0x7f69f8100210>'